packages/selleo-design-core/src/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import { Fragment, h } from "preact";
+import { MenuIcon, SelleoLogo } from "./icons";
+import { CaretDownIcon } from "./icons/CaretDownIcon";
+import cx from "classnames";
+
+type Props = {
+  variant: "desktop" | "mobile";
+};
+
+export function Header({ variant = "desktop" }: Props) {
```

Comment:
i think it can be done with media queries instead of a prop? If it's just for presentation purposes in docs, lets stick to media queries

Reply:
Changed to depend on media queries and created system to present components on mobile layout.